parser_library/src/workspaces/workspace.cpp

```diff
@@ -547,19 +547,22 @@ utils::resource::resource_location transform_to_resource_location(
 {
     utils::resource::resource_location rl;
 
-    if (auto encoded_path = utils::path::encode(path); utils::path::is_uri(encoded_path))
-        rl = utils::resource::resource_location(std::move(encoded_path));
+    if (utils::resource::resource_location(path).is_local())
```

Comment:
I am not questioning whether this works or not, but it seems a bit strange that we construct an object, test it and then we realize that we actually need to construct it a bit differently...

maybe have is_local(string_view) static variant as well?

Reply:
👍